Sources/BugsnagPerformance/Public/BugsnagPerformanceRemoteSpanContext.mm

```diff
@@ -0,0 +1,42 @@
+//
+//  BugsnagPerformanceRemoteSpanContext.mm
+//  BugsnagPerformance-iOS
+//
+//  Created by Robert B on 07/05/2025.
+//  Copyright © 2025 Bugsnag. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <BugsnagPerformance/BugsnagPerformanceRemoteSpanContext.h>
+#import "../Private/Utils.h"
+
+static const int kTraceParentComponentTraceId = 1;
+static const int kTraceParentComponentSpanId = 2;
+
+@implementation BugsnagPerformanceRemoteSpanContext
+
++ (nullable instancetype)contextWithTraceParentString:(NSString *)traceParentString {
+    NSRegularExpression *regex = [NSRegularExpression regularExpressionWithPattern:@"^00-([0-9a-f]{32})-([0-9a-f]{16})-[0-9]{2}$" options:0 error:nil];
```

Comment:
given that we're not using the capturing groups to parse the `traceParentString` maybe we should drop them from the regex?

```suggestion
    NSRegularExpression *regex = [NSRegularExpression regularExpressionWithPattern:@"^00-[0-9a-f]{32}-[0-9a-f]{16}-[0-9]{2}$" options:0 error:nil];
```